src/main/java/net/malisis/core/client/gui/GuiRenderer.java

```diff
@@ -363,8 +395,12 @@ public String clipString(String text, int width)
 		}
 
 		line.append(word);
+		if (prefixColorCode) {
```

Comment:
If there is any remainder and we have a color code, we need to apply it if it isn't a skip line. Should probably check if line isn't empty here (maybe).